pyocd/target/family/target_nRF91.py

```diff
@@ -108,14 +302,104 @@ def split_addr_range_into_chunks(range, chunk_size):
+    0x078c0e01, 0xe004f8c3, 0xeb09d006, 0x2203000c, 0x2102e9c0, 0xbd702065, 0x7efff648, 0x0efff2c0,
+    0x030ceb09, 0x45702402, 0xd906605c, 0x010ceb09, 0xe9c12204, 0x20662002, 0x180bbd70, 0x0501f10e,
+    0x040ceb09, 0x42ab2603, 0xd9066066, 0x000ceb09, 0xe9c02104, 0x20661302, 0x2500bd70, 0x0f91ebb5,
+    0x060ceb09, 0x0504f04f, 0xd0116075, 0xbf000889, 0x68156806, 0xd10542ae, 0x39013004, 0x0204f102,
+    0xe005d1f6, 0x010ceb09, 0x46032206, 0x2002e9c1, 0xbd704618, 0x1030f240, 0x00fff2c0, 0x31016801,
+    0x6800bf1c, 0xf6404770, 0xf2cf71e0, 0x78080100, 0xf3616849, 0x4770200b, 0x2020f240, 0x00fff2c0,
+    0x31016801, 0x6800bf14, 0x5080f44f, 0xbf004770, 0x2024f240, 0x00fff2c0, 0x31016801, 0x6800bf14,
+    0x7000f44f, 0xbf004770, 0x47702000, 0x47702000, 0xf7ffb510, 0x4604ffe1, 0xffeaf7ff, 0xf004fb00,
+    0xbf00bd10, 0x42814401, 0x2001bf9c, 0xbf004770, 0x2b04f850, 0xbf1c3201, 0x47702000, 0xbf244288,
+    0x47702001, 0xbf00e7f4, 0xbf004770, 0x47702000, 0x47702003, 0xbf842803, 0x47702069, 0xb240b580,
+    0xf851a105, 0xf2490020, 0xf2c55104, 0x60080103, 0xf80af000, 0xbd802000, 0x00000000, 0x00000002,
+    0x00000001, 0x00000000, 0x4000f249, 0x0003f2c5, 0x29006801, 0x4770d0fc, 0xf249b580, 0xf2c5500c,
+    0x21010003, 0xf7ff6001, 0xbd80ffef, 0xb082b580, 0x98019001, 0x31fff04f, 0xf7ff6001, 0xb002ffe5,
+    0xbf00bd80, 0x47702069, 0xf7ffb5b0, 0xb148ffa1, 0x25004604, 0xf7ff4628, 0xf7ffffe9, 0x4405ff7d,
+    0xd3f742a5, 0xbdb02000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000
+    ],
+
+    # Relative function addresses
+    'pc_init': 0x20000005,
+    'pc_unInit': 0x20000039,
+    'pc_program_page': 0x200001b5,
+    'pc_erase_sector': 0x200000a9,
+    'pc_eraseAll': 0x20000085,
+
+    'static_base' : 0x20000000 + 0x00000004 + 0x00000448,
+    'begin_stack' : 0x20003460,
+    'end_stack' : 0x20002460,
+    'begin_data' : 0x20000000 + 0x1000,
+    'page_size' : 0x1000,
+    'analyzer_supported' : False,
+    'analyzer_address' : 0x00000000,
+    # Enable double buffering
+    'page_buffers' : [
+        0x20000460,
+        0x20001460
+    ],
+    'min_program_length' : 0x1000,
+
+    # Relative region addresses and sizes
+    'ro_start': 0x4,
+    'ro_size': 0x448,
+    'rw_start': 0x44c,
+    'rw_size': 0x4,
+    'zi_start': 0x450,
+    'zi_size': 0x10,
+
+    # Flash information
+    'flash_start': 0xff8000,
+    'flash_size': 0x1000,
+    'sector_sizes': (
+        (0x0, 0x1000),
+    )
+}
 
 class NRF91(CoreSightTarget):
 
     VENDOR = "Nordic Semiconductor"
 
     def __init__(self, session, memory_map=None):
         super(NRF91, self).__init__(session, memory_map)
+        if memory_map.get_region_for_address(0x00ff8000) is None:
+            LOG.debug("Adding UICR region")
+            memory_map.add_region(FlashRegion(
+            start=0x00ff8000,
+            length=0x1000,
+            blocksize=0x1000,
+            is_testable=False,
+            is_erasable=False,
+            algo=FLASH_ALGO_UICR,
+        ))
```

Comment:
The indentation on this section looks incorrect. The closing parentheses are a level above the indent of the `if` block. Although it will parse ok as is, it should be fixed.